third-party/emissary-ingress/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@kubernetes-models/emissary-ingress",
+  "version": "3.6.0",
```

Comment:
```suggestion
  "version": "0.0.0",
```

Reply:
Oh, should this version not match the version that is provided in the CRDs? That was my understanding of it

Reply:
The version doesn't reflect the CRD version.